src/main/java/frc/robot/subsystems/gripper/Gripper.java

```diff
@@ -94,4 +90,11 @@ public Command getCloseCommand(){
     return new StartEndCommand(this::close, ()-> setPower(0) , this).until(this::getLimSwitchClose);
   }
 
+  @Override
+  public void initSendable(SendableBuilder builder){
+    builder.addBooleanProperty("limSwitch close", this::getLimSwitchClose, null);
+    builder.addBooleanProperty("limSwitch open", this::getLimSwitchOpen, null);
+    SmartDashboard.putData("open gripper",  new StartEndCommand(this::open, ()-> setPower(0) , this).until(this::getLimSwitchOpen));
```

Comment:
```suggestion
    SmartDashboard.putData("Open Gripper",  new StartEndCommand(this::open, ()-> setPower(0) , this).until(this::getLimSwitchOpen));
```

Reply:
Additionally, these commands can't only be in SmartDashboard since they are not used there. Best option is to create a command named Grip that opens and closes. Also, a get function for that command.